revolution_pi/revolution_pi.go

```diff
@@ -12,22 +12,20 @@ import (
 	"sync"
 	"time"
 
-	"github.com/edaniels/golog"
 	commonpb "go.viam.com/api/common/v1"
 	pb "go.viam.com/api/component/board/v1"
 	"go.viam.com/rdk/components/board"
 	"go.viam.com/rdk/grpc"
+	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/resource"
 )
 
-var Model = resource.NewModel("viam-labs", "kunbus", "revolutionpi")
-
 type revolutionPiBoard struct {
 	resource.Named
+	resource.AlwaysRebuild
```

Comment:
What exactly does this do?

Reply:
It always triggers a reconstruction on a config change. In light of your next comment, I think this is the wrong call. What does your config look like? paste it here.

Reply:
Changed it to `triviallyReconfigure`, which is what the old reconfigure function was doing.